superset/async_events/async_query_manager_factory.py

```diff
@@ -32,4 +32,11 @@ def init_app(self, app: Flask) -> None:
         self._async_query_manager.init_app(app)
 
     def instance(self) -> AsyncQueryManager:
+        # check instance init again to allow dynamic feature flag change to apply
+        # after initial app bootstrapping
+        if not self._async_query_manager:
+            from superset import is_feature_enabled
```

Comment:
I believe these imports could be global.

Reply:
It throws `cannot import name 'is_feature_enabled' from partially initialized module 'superset' `